### ISSUE TITLE
PB should have a "only-block-cookies" option

### ISSUE THREAD

**Issue by jgillula** [EFForg/privacybadgerchrome#898]
For folks who aren't savvy enough to unblock entire domains, PB should have a checkbox option under General Settings to switch on "Never block content (Only block cookies)", which would cause PB to only ever block cookies, not entire domains. It should be off by default (i.e. opt-in), and it could have a warning pop up that says "Warning: never blocking content (and only blocking cookies) could leave you exposed to some forms of third-party tracking like pixel tags, [and insert anything else here], which don't use cookies. Are you sure you want to only block cookies? Yes/No"

**Comment by jawz101** [comment]
Browsers already have the ability to block cookies built in.  Seems like if someone just wanted to manage cookies they wouldn't be using Privacy Badger.

**Comment by cowlicks** [comment]
I agree with @jawz101 on this. Maybe instead of "Never block content (Only block cookies)" it should be "Block third party cookies". If they really want the former, they don't really need PB for anything. And there are cases where the user probably *would* want to block content, like google analytics.

If we want to block all third party cookies, we should probably just let the built in browser setting handle this. But I don't know if there is cross platform API to do that. It looks like it might just be available on windows from this: https://developer.chrome.com/extensions/settings_override

**Comment by jgillula** [comment]
Is there a reason not to offer people the option to block only the
third-party cookies that are associated with tracking? Plus, thanks to
the PB interface, people could selectively enable certain third-party
cookies, which is not something you can quickly and easily do via the
browser's built-in UIs.

**Comment by cowlicks** [comment]
Third-party cookies already are blocked from sources associated with tracking, because those sources are not loaded (so they can't even try to set cookies).

That is a good point, about how not using the browser setting would allow users to unblock selected third party domains more easily.

**Comment by cooperq** [comment]
It's worth noting that domains which are "cookie blocked" are actually blocked in hampered from tracking in several ways. In addition to being blocked from setting or receiving cookies they are also blocked from getting referer headers and setting or getting local storage. In the future we hope to increase the number of tracking vectors which are blocked. So this feature would actually still be an improvement over just blocking third party cookies.
